crates/red_knot_python_semantic/resources/mdtest/diagnostics/unpacking.md

```diff
@@ -8,14 +8,20 @@
 a, b = 1  # error: [not-iterable]
 ```
 
-## Too many values to unpack
+## Exactly too many values to unpack
```

Comment:
Is the title change here intentional?

Reply:
Yeah, because I added another test here to cover a diagnostic for "too many values, but not exactly too many" that wasn't previously covered.